include/superior_mysqlpp/converters/to_integer.hpp

```diff
@@ -60,16 +103,31 @@ namespace SuperiorMySqlpp { namespace Converters
             }
         };
 
-
+        /**
+         * Eliminate undesired general conversion.
+         */
         template<typename T, bool validate>
         struct ToIntegerUnsignedImpl
         {
             static inline T call(const char* str, unsigned int length) = delete;
         };
 
+        /**
+         * Converts string into uint8_t.
+         * 
+         * @tparam validate Check if string is convertible.
+         */
         template<bool validate>
         struct ToIntegerUnsignedImpl<std::uint8_t, validate>
         {
+            /**
+             * Parses string to uint8_t.
+             * 
+             * @param str String to be converted.
+             * @param length Length of the string.
+             * @throws std::runtime_error exception if length is not between 1 and 3
```

Comment:
To be consistent with other lines, it should end by '.' character as a sentence.